src/ui/scss/component/_file-list.scss

```diff
@@ -0,0 +1,111 @@
+.file-list__header {
+  display: flex;
+  align-items: center;
+  padding: var(--spacing-vertical-medium);
+  background-color: lighten($lbry-black, 10%);
+  color: $lbry-white;
+  border-top-left-radius: var(--card-radius);
+  border-top-right-radius: var(--card-radius);
+
+  height: 4rem;
+
+  h1,
```

Comment:
This probably shouldn't be an h1

Reply:
Even if it is an `h1`, just give it a class like `file-list__header-text` or whatever